src/Game/UI/Gumps/ShopGump.cs

```diff
@@ -447,6 +439,30 @@ public override bool Draw(UltimaBatcher2D batcher, int x, int y)
             return base.Draw(batcher, x, y);
         }
 
+        private void ProcessListScroll()
+        {
+            if ((DateTime.Now - _lastMouseEventTime).TotalMilliseconds >= SCROLL_DELAY)
```

Comment:
repleace `DateTime.Now` with the `Time.Ticks`
The `DateTime` struct is not necessary here and creates to much overheads under the hood

Reply:
done